### ISSUE TITLE
How to pass binary directly?

### ISSUE THREAD

**Issue by ra1u** [flamegraph-rs/flamegraph#209]
From first example in [README](https://github.com/flamegraph-rs/flamegraph#examples)

    flamegraph /usr/bin/yes
    
would not expect to get error

        error: Found argument '/usr/bin/yes' which wasn't expected, or isn't valid in this context
        
        USAGE:
            flamegraph [OPTIONS] [-- <TRAILING_ARGUMENTS>...]
        
        For more information try --help

**Comment by djc** [comment]
Note the `--` before TRAILING_ARGUMENTS...

**Comment by ra1u** [comment]
@djc in example there is

    # if you'd like to profile an arbitrary executable:
    flamegraph [-o my_flamegraph.svg] /path/to/my/binary --my-arg 5
   
I do not want to post arguments, but path to binary

**Comment by djc** [comment]
Yeah, it seems unfortunately you still need to prefix the arguments with `--` for now. Would be nice to get that fixed but it's not obvious to me right now how.
